obstacle_range: 3.0
raytrace_range: 3.5

# was footprint: [[-0.205, -0.155], [-0.205, 0.155], [0.077, 0.155], [0.077, -0.155]]
robot_radius: 0.17
footprint_padding: 0.05

inflation_radius: 0.75 # was 1.0, 0.2
cost_scaling_factor: 5.0 # was 3.0, 0.1
 
map_type: costmap
observation_sources: scan
scan: {sensor_frame: base_scan, data_type: LaserScan, topic: scan, marking: false, clearing: true}
